src/protojson/UberJobReadyMsg.cc

```diff
@@ -121,7 +168,7 @@ json UberJobReadyMsg::toJson() const {
     jsJr["fileUrl"] = _fileUrl;
     jsJr["rowCount"] = _rowCount;
     jsJr["fileSize"] = _fileSize;
-    return jsJr;
+    return jsJrReqPtr;
```

Comment:
It's simple to return a value rather than the shared pointer:
```
json UberJobReadyMsg::toJsonPtr() const {
    return json::object({
        {"fileUrl", fileUrl},
        {"rowCount", _rowCount},
        {"fileSize", _fileSize;}
    });
 }
```
There won't be any performance penalty due to the value (by move) return optimization

Reply:
The original idea was to just store copies of the json messages and resend them. It got messy, so I dropped it, but using pointers allowed me to send the message and keep the same message in the map without extra copies. Removing the pointers does make the code simpler.